plugins/agent-utilities/skills/fusion-parametric-design/src/fusion_design/reconstruction_program.py

```
@@ -1432,6 +1490,212 @@ def _fillet_archetype(
+                    f"{slab['station_lo']:.6g} and {slab['station_hi']:.6g} on datum {datum_plane}; "
+                    f"its section closes {len(slab['loops'])} loop(s), constant across "
+                    f"{1 + len(slab['constancy']['fractions'])} stations."
+                ),
+            }
+        )
+
+    # A slab whose own measurement failed is not built: its regions come back
+    # with the gate that stopped it, and the slabs above it still stack because
+    # each one's profile is the material measured at its own station.
+    surviving: list[dict[str, Any]] = []
+    for group, slab in zip(planned, slabs):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)